extra/lib/plausible_web/live/customer_support/live/user.ex

```diff
@@ -148,4 +198,24 @@ defmodule PlausibleWeb.CustomerSupport.Live.User do
         {:noreply, assign(socket, form: to_form(changeset))}
     end
   end
+
+  defp keys_query(user) do
+    team_ids = user |> Plausible.Teams.Users.teams() |> Enum.map(& &1.id)
+
+    from api_key in Plausible.Auth.ApiKey,
+      where: api_key.user_id == ^user.id or api_key.team_id in ^team_ids,
```

Comment:
this is going to list all teams' keys as well I think. Just filtering by user should be enough, because
keys and have either:

- user _and_ team set (new keys)
- user set and team set as nil (legacy keys)

```suggestion
      where: api_key.user_id == ^user.id,
```

Reply:
ah, right, OK. thanks 88ad833f6f